src/scripts.js

```diff
@@ -2,32 +2,83 @@ let recipeRepository;
 
 const recipeCarousel = document.querySelector('.recipe-carousel')
 const searchBox = document.querySelector('.search-box');
+const allRecipesButton = document.querySelector('.all-recipes');
+const allRecipesPage = document.querySelector('.all-recipes-page');
+const searchPage = document.querySelector('.search-page')
+const homePage = document.querySelector('.home-page')
+const pageTitle = document.querySelector('.page-title')
+
+const instructionCardDirections = document.querySelector('.instruction-card-directions')
+
+
+window.addEventListener('load', compileRecipeRepository);
+window.addEventListener('load', populateRecipeCarousel);
+
+recipeCarousel.addEventListener('click', () => loadRecipeCard(event))
+searchPage.addEventListener('click', () => loadRecipeCard(event))
 
-window.addEventListener('load', compileRecipeRepository)
-window.addEventListener('load', populateFeaturedRecipesCarousel);
 document.addEventListener('keydown', searchAllRecipes);
+allRecipesButton.addEventListener('click', () => loadSearchPage(recipeRepository.recipes))
+pageTitle.addEventListener('click', loadHomePage)
+
+function loadHomePage() {
+  homePage.classList.remove('hidden')
+  searchPage.classList.add('hidden')
+}
+
```

Comment:
We can reuse this function since its used a couple times
`function loadNewPage(pageTo, pageFrom) {
   pageTo.classList.remove('hidden');
   pageFrom.classList.add('hidden')
}`